pkg/registry/registry.go

```diff
@@ -269,13 +279,11 @@ func (r *Registry) handleMirror(rw mux.ResponseWriter, req *http.Request, ref re
 			proxy := httputil.NewSingleHostReverseProxy(u)
 			proxy.Transport = r.transport
 			proxy.ErrorHandler = func(_ http.ResponseWriter, _ *http.Request, err error) {
-				log.Error(err, "proxy failed attempting next")
+				log.Error(err, "mirror request failed attempting next")
 			}
 			proxy.ModifyResponse = func(resp *http.Response) error {
 				if resp.StatusCode != http.StatusOK {
-					err := fmt.Errorf("expected mirror to respond with 200 OK but received: %s", resp.Status)
-					log.Error(err, "mirror failed attempting next")
-					return err
+					return fmt.Errorf("expected mirror to respond with 200 OK but received: %s", resp.Status)
```

Comment:
I think this message would be somewhat clearer if it said "peer" rather than "mirror".

Reply:
I do not agree with this. The word peer is something that has confused people i other issues. While peer is fine to use in code I would rather use the word mirror externally. It is a word that more people understand in "normal" image mirroring setups.